test_cover.sh

```diff
@@ -0,0 +1,26 @@
+#/bin/sh
+# http://www.songmu.jp/riji/entry/2015-01-15-goveralls-multi-package.html
+set -e
+# 中断した時とかにゴミが残らないようにcleanupする
```

Comment:
please write comment in English (e.g. clean-up if interrupt the process)
L15 is same